rust/catalyst-contest/Cargo.toml

```diff
@@ -15,14 +15,19 @@ workspace = true
 [dependencies]
 anyhow = "1.0.100"
 async-trait = "0.1.89"
+
+# TODO: Use tag instead of path.
+catalyst-signed-doc = { version = "0.0.10", path = "../signed_doc" }
+catalyst-voting = { version = "0.0.2", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "catalyst-voting/v0.0.2" }
+
+cbork-utils = { version = "0.0.3", git = "https://github.com/input-output-hk/catalyst-libs.git", tag = "cbork-utils/v0.0.3.1" }
```

Comment:
I know that its a small update, but lets merge [feat/blake3-hash-cbor-tag](https://github.com/input-output-hk/catalyst-libs/tree/feat/blake3-hash-cbor-tag) it into the `main` with bumping `cbork-utils` crate version and make a proper release.

Reply:
This crate now points to the `cbork-utils/v0.0.4` release.